configs/wikitext/wikitext_language_modeling_seq2seq.yml

```diff
@@ -94,14 +96,17 @@ pipeline:
 
   # LSTM Decoder
   lstm_decoder:
-    type: RecurrentNeuralNetwork
+    type: Attn_Decoder_RNN
     priority: 4
-    initial_state: Input
-    hidden_size: 300
-    num_layers: 3
+    hidden_size: 50
+    num_layers: 1
     use_logsoftmax: False
+<<<<<<< Updated upstream
```

Comment:
Hmmmm, I guess the "updated upstream" is the right one... but not sure..

Reply:
I guess I got mixed up with my stashes. I checked-out the `develop` version of the two files in question, and that solved it.